docs/HACKING.md

```diff
@@ -167,37 +310,21 @@ dotrun
 
 # MAAS deployments
 
-## Canonical VPN deployments
-
-If you have access to the Canonical VPN you can use one of the following MAAS deployments. You may need to do some [additional configuration](#vpn-configuration) inside your multipass instance.
-
-Once connected to the VPN you can connect to one of the following MAAS deployments using the [credentials](https://wiki.canonical.com/WebAndDesign/DesignMaasLab).
-
-### Karura
-
-[karura.internal](http://karura.internal:5240/MAAS) (last stable release).
-
-### Bolla
-
-[bolla.internal](http://bolla.internal:5240/MAAS) (main)
-
-## Local deployments
-
-### Snap deployment
+## Snap deployment
 
 The easiest way to run a MAAS locally is using a snap. However, this method does not provide sample data and therefore will not have everything e.g. there will be no machines.
 
 First you'll need to either [create a Multipass instance](#creating-a-multipass-instance) or [create a LXD container](#creating-a-lxd-container), call it something like "snap-maas".
 
 Then enter the shell for that instance:
 
-#### Multipass
+### Multipass
 
 ```shell
 multipass shell snap-maas
 ```
 
-#### LXD
+### LXD
 
 ```shell
 lxc exec snap-maas bash -- su ubuntu
```

Comment:
This command doesn't need `bash` to be called explicitly, you can just do `lxc exec snap-maas -- su ubuntu`